src/api.js

```diff
@@ -0,0 +1,82 @@
+const axios = require('axios');
+const rateLimit = require('axios-rate-limit');
+require('dotenv').config({path: '../.env'})
+const { ETHERSCAN_API_KEY, ETHERSCAN_API_URL, FAUCET_ADDRESS, GOERLI_API_URL } = process.env;
+
+const getBlockNumber = async function(time) {
+    const request = await rateLimit(axios.create(), {maxRequests: 5, perMillisecondss: 1000, maxRPS: 1})
```

Comment:
Not necessary to allocate memory using `axios.create()` and not necessary to build `request` object on every `getBlockNumber`, `getTransactions` and `getBalance` function calls.
Probably better is to build request object once on a file scope and re-use it.